test/unittests/validation_stack_test.cpp

```diff
@@ -548,8 +548,8 @@ TEST(validation_stack, if_invalid_end_stack_height)
     */
     const auto wasm = from_hex(
         "0061736d01000000010401600000030201000a1701150042014102047e4201420205420342041a0b1a1a0b");
-    parse(wasm);
-    // TODO: Add max stack height check.
+    const auto module = parse(wasm);
+    EXPECT_EQ(module.codesec[0].max_stack_height, 3);
```

Comment:
is the comment on line 537 outdated?

Reply:
No, it explains `--no-check` and why this is invalid wasm. But we don't have "too many results" validation check yet (there is a placeholder for it added in #319). We will need to modify this test's expectations at some point later.